integration_tests/env_variable_names.py

```diff
@@ -35,3 +35,8 @@
 SLACK_SDK_TEST_INCOMING_WEBHOOK_CHANNEL_NAME = (
     "SLACK_SDK_TEST_INCOMING_WEBHOOK_CHANNEL_NAME"
 )
+
+# For Slack Connect shared tests
+SLACK_SDK_TEST_SENDER_BOT_TOKEN = "SLACK_SDK_TEST_SENDER_BOT_TOKEN"
```

Comment:
Can we make these names more specific?
```suggestion
SLACK_SDK_TEST_CONNECT_INVITE_SENDER_BOT_TOKEN = "SLACK_SDK_TEST_CONNECT_INVITE_SENDER_BOT_TOKEN"
```